<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPointCloudObjectCreation</class>
 <widget class="QDialog" name="DialogPointCloudObjectCreation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Point Cloud Object from box</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_file">
     <property name="text">
      <string>Filename</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_density">
     <property name="text">
      <string>Decimation</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDoubleEdit" name="lineEdit_density"/>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_density_unit">
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_import">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="QRegExpEdit" name="lineEdit_file">
     <property name="placeholderText">
      <string>Enter filename</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QRegExpEdit</class>
   <extends>QLineEdit</extends>
   <header>regexpedit.h</header>
  </customwidget>
  <customwidget>
   <class>QDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>qdoubleedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_file</tabstop>
  <tabstop>lineEdit_density</tabstop>
  <tabstop>pushButton_import</tabstop>
  <tabstop>pushButton_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
